core/controllers/payload_validator.py

```diff
@@ -94,6 +94,7 @@ def validate(handler_args, handler_args_schemas, allowed_extra_args):
     'ContributorRightsDataHandler',
     'CreatorDashboardHandler',
     'CreatorDashboardPage',
+    'CronTranslationContributionStatsHandler',
```

Comment:
Undo changes in this file, we are not expecting to add new handlers in this list*